admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
+| `/settlementModels/{name}`  | View settlement model by name  | Update a settlement model (activate/deactivate a settlement model) | Not supported  |  Not supported |
+| **`/transactions/{id}`**  | Retrieve transaction details by `transferId`  | Not supported  | Not supported  | Not supported |
+
+## API Resource `/participants`
+
+The services provided by the resource `/participants` are primarily used by the Hub Operator for viewing, creating, and updating participant-related details, such as limit (Net Debit Cap), position, or endpoints configured.
+
+### `GET /participants`
+
+The `GET /participants` request is used to retrieve information about all participants.
+
+#### Data model
+
+The `GET /participants` request sends an empty request body.
+
+The following data is returned for each participant in the case of a successful operation:
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `name`  |  yes | [String(2..30)](#string) | The name of the participant. |
+| `id`  |  yes | [String](#string) | The identifier of the participant in the form of a fully qualified domain name combined with the participant's `fspId`. |
+| `created`  |  yes | [DateTime](#datetime)  | Date and time when the participant was created. |
+| `isActive`  |  yes | [Integer(1)](#integer) | A flag to indicate whether or not the participant is active. Possible values are `1` and `0`. |
+| `links`  |  yes | [Self](#self) | List of links for a Hypermedia-Driven RESTful Web Service. | 	
+| `accounts`  |  yes | [Accounts](#accounts) | The list of ledger accounts configured for the participant. |
+
+### `POST /participants`
+
+The `POST /participants` request is used to create a participant in the Hub.
+
+#### Data model
+
+The `POST /participants` request sends the following data in the request body:
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `name`  |  yes | [String(2..30)](#string) | The name of the participant. |
+| `currency`  |  yes | [CurrencyEnum](#currencyenum) | The currency that the participant will transact in. |
+
+The following data is returned for the newly created participant in the case of a successful operation:
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `name`  |  yes | [String(2..30)](#string) | The name of the participant. |
+| `id`  |  yes | [String](#string) | The identifier of the participant in the form of a fully qualified domain name combined with the participant's `fspId`. |
+| `created`  |  yes | [DateTime](#datetime)  | Date and time when the participant was created. |
+| `isActive`  |  yes | [Integer(1)](#integer) | A flag to indicate whether or not the participant is active. Possible values are `1` and `0`. |
+| `links`  |  yes | [Self](#self) | List of links for a Hypermedia-Driven RESTful Web Service. | 	
+| `accounts`  |  yes | [Accounts](#accounts) | The list of ledger accounts configured for the participant. |

```

Comment:
Let's cut down on boilerplate language in favour of consistent headings.

```suggestion

##### Parameters

| Name  | Required  | Type | Description |
|---|---|--|--|
| `name`  |  yes | [String(2..30)](#string) | The name of the participant. |
| `currency`  |  yes | [CurrencyEnum](#currencyenum) | The currency that the participant will transact in. |

##### Returns
A newly created participant.

| Name  | Required  | Type | Description |
|---|---|--|--|
| `name`  |  yes | [String(2..30)](#string) | The name of the participant. |
| `id`  |  yes | [String](#string) | The identifier of the participant in the form of a fully qualified domain name combined with the participant's `fspId`. |
| `created`  |  yes | [DateTime](#datetime)  | Date and time when the participant was created. |
| `isActive`  |  yes | [Integer(1)](#integer) | A flag to indicate whether or not the participant is active. Possible values are `1` and `0`. |
| `links`  |  yes | [Self](#self) | List of links for a Hypermedia-Driven RESTful Web Service. | 	
| `accounts`  |  yes | [Accounts](#accounts) | The list of ledger accounts configured for the participant. |

```

... and so on...

Reply:
I don't agree with the "Parameters" bit. I would prefer to keep the term "parameter" for query parameters or path parameters (that practice is also in sync with Swagger terminology). Having said that, I _will_ cut down on boilerplate language.

Reply:
Sure. We could have separate table for query or path parameters, or some specs have a `where` column in the table which states where the parameter should be places (path, query, body etc.) 

Mainly I though we could do better than vague terms such as: _"sends the following data "_